plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -402,6 +428,12 @@ static void nvenc_defaults(obs_data_t *settings)
 	obs_data_set_default_bool(settings, "2pass", true);
 	obs_data_set_default_int(settings, "gpu", 0);
 	obs_data_set_default_int(settings, "bf", 2);
+	obs_data_set_default_bool(settings, "spatialaq", false);
+	obs_data_set_default_bool(settings, "temporalaq", false);
+	obs_data_set_default_bool(settings, "forcedidr", false);
+	obs_data_set_default_int(settings, "spatialaqstrength", 1);
```

Comment:
The default setting for `aq-strength` (spatialaqstrength) should be 8, according to FFmpeg.